WalletWasabi.Fluent/Models/UI/UIContext.cs

```diff
@@ -0,0 +1,18 @@
+using Avalonia;
+using Avalonia.Input.Platform;
+
+namespace WalletWasabi.Fluent.Models.UI;
+
+public class UIContext
+{
+	public UIContext(IQrCodeGenerator qrCodeGenerator, IClipboard clipboard)
+	{
+		QrCodeGenerator = qrCodeGenerator;
+		Clipboard = clipboard;
+	}
+
+	public IClipboard Clipboard { get; }
+	public IQrCodeGenerator QrCodeGenerator { get; }
+
+	public static UIContext Default => new(new QrGenerator(), Application.Current?.Clipboard);
```

Comment:
In the case of `Application.Current` is null, null would be passed to `UIContext` ctor and `IClipboard` is not marked as nullable, so it could trick us.

In what scenario can the `Application.Current` be null? Can it even affect us?

Reply:
@soosr `Application.Current` will be null if this code is running in a unit test.

This is why, as explained [here](https://github.com/zkSNACKs/WalletWasabi/pull/10308), `UIContext` is actually a dependency of every Viewmodel, whether it requires to do navigation, use clipboard, or any other external dependencies.

Also explained there as well, the `UIContext.Default` property is just temporary, intended as an intermediate workaround until we finalize the refactoring of all Viewmodels.

IMO I would keep it like this by now. Marking the `Clipboard` property as nullable will only add unnecessary null checks in many places when in practice it can't really be null (because unit tests will use a mocked UIContext with required dependencies properly initialized)

Reply:
> IMO I would keep it like this by now. Marking the Clipboard property as nullable will only add unnecessary null checks in many places when in practice it can't really be null (because unit tests will use a mocked UIContext with required dependencies properly initialized)

Yes, this makes sense.
Can you add a comment about it is a temporary solution, and eliminated the null warning with a `!`?

Reply:
Done